layouts/partner/aws.html

```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
+
+            <div class="w-full lg:w-1/2 px-8">
+                <p class="text-3xl font-bold inline-block">AWS Crosswalk</p>
+                <p class="mt-0">
+                    Pulumi Crosswalk for AWS is a collection of libraries that use automatic
+                    well-architected best practices to make common infrastructure-as-code
+                    tasks in AWS easier and more secure. Secure and cost-conscious defaults are chosen
+                    so that simple programs automatically use best practices for the underlying
+                    infrastructure, enabling better productivity with confidence.
+                </p>
+
+                <p class="text-lg font-bold">Application and Infrastructure Together</p>
+                <p>
+                    Crosswalk enables you to blur the lines between application and infrastructure code enabling
+                    you to author an entire full stack application in one program. With support for inline Lambda
+                    functions and ease-of-use helper functions, building robust applications on AWS has never
+                    been easier.
+                </p>
+                <p class="my-4 mt-8 text-left w-full">
+                    <a class="inline-flex items-center text-gray-700 hover:text-gray-800 font-bold text-xl" href="{{ relref . "/docs/guides/crosswalk/aws" }}">
+                        <span class="inline-block w-6 text-orange-300 mr-3">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="sparkles" class="svg-inline--fa fa-sparkles fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g class="fa-group"><path class="fa-secondary" fill="currentColor" d="M423.16 186.58L448 127l59.58-24.84a8 8 0 0 0 0-14.32L448 63 423.16 3.42a8 8 0 0 0-14.32 0L384 63l-59.58 24.84a8 8 0 0 0 0 14.32L384 127l24.84 59.58a8 8 0 0 0 14.32 0zm-14.32 136.84L384 383l-59.58 24.84a8 8 0 0 0 0 14.32L384 447l24.84 59.58a8 8 0 0 0 14.32 0L448 447l59.58-24.84a8 8 0 0 0 0-14.32L448 383l-24.84-59.58a8 8 0 0 0-14.32 0z" opacity="0.4"></path><path class="fa-primary" fill="currentColor" d="M384 254.64a16.06 16.06 0 0 0-8.84-14.33l-112.57-56.39-56.28-112.77c-5.44-10.87-23.19-10.87-28.62 0l-56.28 112.77L8.84 240.31a16 16 0 0 0 0 28.67l112.57 56.39 56.28 112.77a16 16 0 0 0 28.62 0l56.28-112.77L375.16 269a16.07 16.07 0 0 0 8.84-14.36z"></path></g></svg>
+                        </span>
+                        <span class="mr-2 hover:mr-3 transition-all">
+                            Learn More About Crosswalk
+                        </span>
+                        <span>
+                            &rarr;
+                        </span>
+                    </a>
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <!-- Start AWS re:Invent takeover content -->
+    <section id="workshops" class="container mx-auto text-center mb-12">
+        <p class="text-3xl font-bold">JOIN A WORKSHOP</p>
```

Comment:
Shouldn't this be a heading (e.g., `h3`)?

Reply:
👍